e2e/test/scenarios/dashboard-filters/reproductions/31662-between-filter-default-value.cy.spec.js

```diff
@@ -29,10 +29,14 @@ describe("issue 31662", () => {
       },
     );
     cy.findByTestId("dashboard-empty-state").should("be.visible");
-    filterWidget().findByText("2 selections").click();
+    editDashboard();
+    cy.findByTestId("edit-dashboard-parameters-widget-container")
+      .findByText("Between")
+      .click();
+    sidebar().findByText("2 selections").click();
     popover().within(() => {
-      cy.findByDisplayValue("10").should("be.visible");
-      cy.findByDisplayValue("20").should("be.visible");
+      cy.findByDisplayValue("3").should("be.visible");
+      cy.findByDisplayValue("5").should("be.visible");
```

Comment:
I'm not sure I understand what you're suggesting @WiNloSt. What I wanted to do here is to ensure we keep the filter states while switching between tabs

Reply:
If I remember correctly, on master we're testing the actual filter values, but this test in this PR is changed so that we're asserting the default value in the filter settings instead as this dashboard doesn't have the filter mapped to a card. Please see the screenshot below.
![image](https://github.com/metabase/metabase/assets/1937582/186bb740-97d1-4cfc-a226-6e6320da31c3)
![image](https://github.com/metabase/metabase/assets/1937582/66e17890-e50b-4840-8725-dd70e6fec2f4)